providers/music/songkick.go

```diff
@@ -0,0 +1,89 @@
+## Feel free to ignore this file 
```

Comment:
Ignoring the bulk of this but just wanted to drop the suggestion again for the `NewRequest()` model -
```go
func (c *Client) NewRequest(method, urlStr string, body io.Reader) (*http.Request, error) {
```

Reply:
Let's discuss f2f because I'm thinking of punting on templating the endpoint and method since we only have 1 use case for this api?